PERFORMANCE.MD

```diff
@@ -0,0 +1,65 @@
+# OpenCensus Service Performance
+
+The performance numbers that follow were generated using version 0.1.3 of the
+OpenCensus Service, are applicable primarily to the OpenCensus Collector and
+are measured only for traces. In the future, more configurations will be tested.
+
+Note with the OpenCensus Agent you can expect as good if not better performance
+with lower resource utilization. This is because the OpenCensus Agent does not
+today support features such as batching or retries and will not support
+tail-sampling.
+
+It is important to note that the performance of the OpenCensus Collector depends
+on a variety of factors including:
+
+* The receiving format: Jaeger thrift (14268) or Zipkin v2 JSON (9411)
+* The size of the spans (tests are based on number of tags): 20
+* Whether tail-sampling is enabled or not
+* CPU / Memory allocation
+
+## Testing
+
+Testing was completed using the [Synthetic Load Generator utility](https://github.com/Omnition/synthetic-load-generator) and results can
+be reproduced in your own environment. It is important to note that this utility
+has a few configurable parameters which can impact the results of the tests. The
+parameters used are defined below.
+
+* FlushInterval(ms) [default: 1000]
+* MaxQueueSize [default: 100]
+* SubmissionRate(spans/sec): 100,000
+
+## Results without tail-based sampling
+
+| Span<br>Format    | CPU<br>(2+ GHz) | RAM<br>(GB) | Sustained<br>Rate | Recommended<br>Maximum |
```

Comment:
1. Can we please add numbers with our own format? :)
1. What is the exporter in all these case?

Reply:
1. We still need to change the synthetic-load-generator to emit oc (although we could measure oc-agent -> oc-collector)
2. In this case we are interested on ingress rate so we measure with a "null exporter" (i.e.: --debug-processor)

Reply:
We can do the following:

(1)load generator (import_format) -> (2)"test_instance" (export_fromat) -> (3)oc_agent with debug

where:
(1) is the load generator
(2) is the instance we monitor
(3) acts just as a backend (in reality they may have issues with scaling the backend, but this is beyond this project scope).

Reply:
In order to get these numbers published today we will skip, but the goal is to include these going forward (ideally by using the Java ocagent exporter)

Reply:
Please define what sustained rate means

Reply:
@yurishkuro done: https://github.com/census-instrumentation/opencensus-service/pull/441

Reply:
sorry, still not clear, that change just says "running for a minimum of one hour (i.e. sustained rate)". Rate of what? Number of spans received by the collector? What does it do with the spans besides receiving them?